src/website/webview/static/style.css

```
@@ -850,6 +850,56 @@ a:focus {
   text-decoration: underline;
 }
 
+.maintainers-list {
+  display: flex;
+  flex-direction: column;
+  margin-left: 1em;
+}
+
+.maintainer-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 2em;
+}
+
+.maintainer-container input {
+  margin: 0;
+  appearance: none;
+  width: 2em;
+  height: 2em;
+  cursor: pointer;
+}
+
+.maintainer-container input::after {
+  position: relative;
+  top: -0.2em;
+  content: "↺";
+  text-align: center;
+  font-weight: 700;
+  font-size: 1.5em;
+  color: var(--grey);
+}
+
+.maintainer-container input:checked::after {
+  position: relative;
+  top: 0.2em;
+  content: "🗑";
+  text-align: center;
+  font-size: 1.2em;
+  color: var(--dark-grey);
+}
+
+.maintainer-container input ~ article {
+  margin: 0;
+  padding: 0;
+}
+
+.maintainer-container input:not(:checked) ~ article {
```

```suggestion
.maintainer-container input:not(:checked) ~ article,
.maintainer-container input:not(:checked) ~ article a.github-id
{
```

So the handle also gets greyed out